README.md

```diff
@@ -63,8 +63,9 @@ Create a discord user and join the server you want to connect to IRC.
     },
     "ircOptions": { // Optional node-irc options
       "floodProtection": false, // On by default
-      "floodProtectionDelay": 1000 // 500 by default
+      "floodProtectionDelay": 1000, // 500 by default
     },
+    "ircNickColor": false, // Gives usernames a color in IRC for better readability (on by default)
```

Comment:
Maybe this should be an object, so that you could toggle colors for both IRC and Discord?
Something like this:
```js
"nickColors": {
  "irc": true,
  "discord": false
}
```

Or maybe only one option that covers both is necessary? I.e. something like `nickColors`.